src/titiler/application/titiler/application/main.py

```diff
@@ -58,7 +58,7 @@
         CORSMiddleware,
         allow_origins=api_settings.cors_origins,
         allow_credentials=True,
-        allow_methods=["GET"],
+        allow_methods=["GET", "POST"],
```

Comment:
If this is missing, the browser's CORS request to /mosaicjson/mosaics always fails